.asf.yaml

```diff
@@ -0,0 +1,17 @@
+github:
+  features:
+    issues: true
+  enabled_merge_buttons:
+    merge: false
+    rebase: true
+    squash: true
+  dependabot_alerts: true
+  dependabot_updates: false
+  protected_branches:
+    main:
```

Comment:
Do we want to require a linear history, too? I really think having commits be signed would be a huge plus when it comes to IP as well, though I can understand people not wanting it.

Reply:
I think such a

Reply:
+1